.env.example

```
@@ -0,0 +1,21 @@
+# Copy to .env. Keep all real values out of version control.
+MERIDIAN_TELLER_OPERATOR=teller1
+MERIDIAN_TELLER_PASSWORD=
+MERIDIAN_SUPERVISOR_OPERATOR=super1
+MERIDIAN_SUPERVISOR_PASSWORD=
+MERIDIAN_BRANCH=MAIN-001
+# Three distinct random values, each at least 32 characters. Keep the HMAC key stable.
+CALLER_API_TOKEN=
+OPERATOR_API_TOKEN=
+JOURNAL_HMAC_KEY=
+CALLER_CAPABILITIES=meridian-sign-on,meridian-member-inquiry,meridian-member-record,meridian-funds-transfer,meridian-open-share,meridian-update-member,meridian-place-hold
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CALLER_CAPABILITIES key should go before the JOURNAL_HMAC_KEY key

(UnorderedKey)